state-chain/chains/src/address.rs

```diff
@@ -68,6 +73,7 @@ pub enum EncodedAddress {
 	Eth([u8; 20]),
 	Dot([u8; 32]),
 	Btc(Vec<u8>),
+	Arb([u8; 20]),
```

Comment:
^ see prev comment, we have an `EncodedAddress::Arb` - if they're treated the exact same then I'd question whether we need one - we could just make it ::Evm?